sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -1032,9 +1032,19 @@ object Hex {
     array
   }
 
+  private def maxArraySize(length: Long): Int = {
```

Comment:
Since this is not reused at all, shall we inline the logic instead of making a new method?

Reply:
Ok, thank you @dongjoon-hyun and welcome back